src/util/webhooks.ts

```diff
@@ -0,0 +1,59 @@
+import statsd from "../config/statsd";
+
+export const getCurrentTime = () => Date.now();
+
+export const calculateProcessingTimeInSeconds = (
```

Comment:
I'd rather call it `emitWebhookProcessingTimeMetrics`

Also, if we don't use the return value, we should make it Void

Reply:
Am I not already doing that?  number | undefined | void